source/deployment/cluster.rst

```diff
@@ -134,75 +134,143 @@ A sample configuration for NGINX is provided below.  It assumes you have two Mat
     }
 
 
-A setup with multiple proxy servers can be utilized to limit a single point of failure, but is beyond the scope of this documentation.
-
+You can use multiple proxy servers to limit a single point of failure, but that is beyond the scope of this documentation.
 
 File Storage Configuration
-----------------------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. note:: File storage is assumed to be shared between all the machines utilizing services such as NAS or Amazon S3. If ``"DriverName": "local"`` is used then the directory at ``"FileSettings":`` ``"Directory": "./data/"`` is expected to be a NAS location mapped as a local directory, otherwise high availability will not function correctly and may corrupt your file storage. If you’re using Amazon S3 for file storage then no other configuration is required.
+.. note::
+  1. File storage is assumed to be shared between all the machines that are using services such as NAS or Amazon S3.
+  2. If ``"DriverName": "local"`` is used then the directory at ``"FileSettings":`` ``"Directory": "./data/"`` is expected to be a NAS location mapped as a local directory, otherwise high availability will not function correctly and may corrupt your file storage.
+  3. If you’re using Amazon S3 for file storage then no other configuration is required.
 
-If you’re using the Compliance Reports feature in Enterprise Edition E20, you will need to configure the  ``"ComplianceSettings":`` ``"Directory": "./data/",`` to share between all machines or the reports will only be available from the System Console on the local Mattermost server.
+If you’re using the Compliance Reports feature in Enterprise Edition E20, you need to configure the  ``"ComplianceSettings":`` ``"Directory": "./data/",`` to share between all machines or the reports will only be available from the System Console on the local Mattermost server.
 
 Migrating to NAS or S3 from local storage is beyond the scope of this document.
 
 Database Configuration
-------------------------------------
-Scaling the database can be accomplished by utilizing the read-replica feature. The Mattermost server can be set up to use one "master" database and up to 8 read replica databases. Mattermost distributes read requests across all databases, and sends write requests to the master database, and those changes are then sent to update the read replicas. 
+^^^^^^^^^^^^^^^^^^^^^^
+
+Use the read-replica feature to scale the database. The Mattermost server can be set up to use one "master" database and up to 8 read replica databases. Mattermost distributes read requests across all databases, and sends write requests to the master database, and those changes are then sent to update the read replicas. 
```

Comment:
"and up to 8 read replica" is a limit of Aroua (I think).  Maybe "and multiple read replica"